OKR.md

```diff
@@ -0,0 +1,50 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### The daemon is stable and so core is reliable
+
+- `PX` - OWNER - The uncaught errors and memory usage are under control and a daemon can run for a week non stop
+- `PX` - OWNER - One or more js-ipfs daemons are part of the IPFS infrastructure
+- `PX` - OWNER - js.ipfs.io exposes an IPFS Gateway provided by a js-ipfs daemon
+
+### The core can handle vast amounts of data
+
+- `PX` - OWNER - The MFS is implemented in js-ipfs
+- `PX` - OWNER - npm on IPFS is used as a test vector of js-ipfs capabilities
+- `PX` - OWNER - js-ipfs in the browser can load a dataset of at least 100Gb (create archives.js.ipfs.io)
+- `PX` - OWNER - js-ipfs in Node.js can load a dataset of at least 1TB (use npm to prove it)
+- `PX` - OWNER - A S3 IPFS Repo backend exists and it is easily pluggable on `new IPFS`
+- `PX` - OWNER - The PIN API is finished and merged.
+
+### The APIs are more ergonomic and developer productivity is off the charts
+
+- `PX` - OWNER - Complete the first milestone of the Awesome Endeavour - Flow
+- `PX` - OWNER - Roadmap and Kickstart the second milestone for the Awesome Endeavour - Flow
+- `PX` - OWNER - Make js-ipfs uglify friendly again
+- `PX` - OWNER - Every env test run is parallelized in CI
+- `PX` - OWNER - CLI, HTTP-API and Core Tests of js-ipfs are parallelized in CI
+- `PX` - OWNER - A repo migration utility exists and migration is a good experience for js-ipfs users
+
+### libp2p levels up, learns new superpowers and a dedicated team is formed
+
+- `PX` - OWNER - libp2p has primitives to keep connections under control
```

Comment:
@pgte did a ton of work in Q1 to make this possible. @pgte wanna continue handling it for Q2?